TrackingAntennaV3/main/Accelerometer.cpp

```diff
@@ -0,0 +1,41 @@
+/**
+ * @file   Magnetometer.cpp
+ * @Author Michael Lenover (WARG)
+ * @date   January 25, 2017
+ */
+
+#include "Logger.hpp"
+#include "Accelerometer.hpp"
+#include <Adafruit_LSM303_U.h>
+
+//Create accelerometer object
+static Adafruit_LSM303_Accel_Unified accel;
+
+//Declare accelerometer event variable
+static sensors_event_t accelEvent;
+
+//Initializes accelerometer struct
+volatile Accelerometer accelerometer;
+
+//Initializes accelerometer
+void initAccelerometer(){
+ //Defines accel object based on compass id
+ accel = Adafruit_LSM303_Accel_Unified(COMPASS_ID);
+
+ //Initializes accelerometer
+ if(!accel.begin()){
+   error("\nAccelerometer failed to be detected.");
```

Comment:
If theres a potential of the sensor not initializing properly, you should also return a boolean (1 or 0) with this function. This will allow us to decide waht happens in the main function (do we try again, wait a little bit then try, etc..)